interpreter/cling/tools/plugins/clad/CMakeLists.txt

```diff
@@ -95,8 +95,8 @@ ExternalProject_Add(
              -DLLVM_DIR=${LLVM_BINARY_DIR}
              -DCLANG_INCLUDE_DIRS=${CLANG_INCLUDE_DIRS}
              ${_clad_extra_cmake_args}
-  BUILD_COMMAND ${CMAKE_COMMAND} --build . ${EXTRA_BUILD_ARGS}
-  INSTALL_COMMAND ${CMAKE_COMMAND} --build . ${EXTRA_BUILD_ARGS} --target install
+  BUILD_COMMAND ${CMAKE_COMMAND} --build . ${EXTRA_BUILD_ARGS} -j 1
```

Comment:
Consider adding a comment why '-j 1' is needed (is it a temporary workaround until something is fixed in clad or is it a limitation of the platform and/or MV configuration?).

Reply:
I agree. I wanted to test this at scale before documenting further. @vgvassilev any thought about why this is now necessary?

Reply:
I am not sure I have seen the actual build failure to be able to comment. Can you give me a link to the failure?

Reply:
Just remove the "-j1"

Reply:
I do not have access to ubu24 system.

Reply:
but you have the line to reproduce the failure locally with the container, right?

Reply:
https://github.com/root-project/root/issues/16733#issuecomment-2435180193